.github/workflows/bootstrap/action.yml

```diff
@@ -0,0 +1,20 @@
+---
+name: Bootstrap Checkout
+description: Ensures an action that needs to build has access to git tags
+
+runs:
+  using: "composite"
+  steps:
+    # Ensure we fetch all tags
+    - shell: bash
+      run: |
+        git fetch --prune --unshallow --tags
```

Comment:
I somehow thought the `checkout: depth: 0` would be enough 

```
    # Number of commits to fetch. 0 indicates all history for all branches and tags.
    # Default: 1
    fetch-depth: ''
```

In https://github.com/actions/checkout